WooCommerce/Classes/ViewRelated/AppCoordinator.swift

```diff
@@ -122,33 +127,62 @@ private extension AppCoordinator {
     /// Displays the WordPress.com Authentication UI.
     ///
     func displayAuthenticator() {
```

Comment:
Nit: should we clarify this method by renaming it to `displayAuthenticatorWithOnboardingIfNeeded`? I think that will differentiate it better from `configureAndDisplayAuthenticator`.

Reply:
Makes sense, renamed in https://github.com/woocommerce/woocommerce-ios/pull/7551/commits/9a518d5df5f5c539abd869d21136b3782c2374dc